modules/logging-loki-retention.adoc

```diff
@@ -46,9 +44,47 @@ spec:
 ----
 <1> Sets retention policy for all log streams. *Note: This field does not impact the retention period for stored logs in object storage.*
 <2> Retention is enabled in the cluster when this block is added to the CR.
-<3> Contains the link:https://grafana.com/docs/loki/latest/logql/query_examples/#query-examples[LogQL query] used to define the log stream.
-+
-.Example per-tenant stream-based retention
+<3> Contains the link:https://grafana.com/docs/loki/latest/logql/query_examples/#query-examples[LogQL query] used to define the log stream.spec:
+  limits:
+ 
+.Example global stream-based retention for GCP
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+metadata:
+  name: logging-loki
+  namespace: openshift-logging
+   global: <1>
+      retention: <2>
+        days: 20
+        streams:
+        - days: 4
+          priority: 1
+          selector: '{kubernetes_namespace_name=~"test.+"}' <3>
+        - days: 1
+          priority: 1
+          selector: '{log_type="infrastructure"}'
+  managementState: Managed
+  replicationFactor: 1
+  size: 1x.small
+  storage:
+    schemas:
+    - effectiveDate: "2020-10-11"
+      version: v11
```

Comment:
It is better to use same version in all Lokistack examples and explain what the version is?  For 5.8 and prior, we can use v11,v12. from 5.9: we can use v13.   @kabirbhartiRH  correct me if I am wrong?

Reply:
Yes, for Logging 5.9 onwards schemas version should be v13. Prior to that, v11 (default) and v12 can be chosen.

Reply:
Both AWS Lokistack examples reference v11. 

Should there be a note that explains that logging for 5.8 and prior, v11 or v12 should be the designated version number, but from 5.9+, it should be v13?

Reply:
5.9+ still supports v12 but v13 is the recommended schema version for 5.9+
It would be a good to have note.

Reply:
Added "Although logging version 5.9 and higher supports schema v12, v13 is recommended." as a note.